some design

karmelp opened the pull request:
proovisin isDonei teha, aga ei saanud hakkama. Seega lisasin eventlisteneriga done button värvimuutuse ja seda, kuidas ma tegelikult peaks seda muutma, tuleb siis, kui aru saan, mismoodi ma saan taski lisades isDone jätta undone ehk false ja selle nupu vajutamisel muuta nupu värvi, tõmmata kriips tekstile peale ja muuta isdone done'iks ja seeläbi lisada see tehtud taskide numbritesse +1'na ja selle nupu uuesti vajutamisel undo kõik, mis just ütlesin.

karmelp committed:
some design

proovisin isDonei teha, aga ei saanud hakkama. Seega lisasin eventlisteneriga done button värvimuutuse ja seda, kuidas ma tegelikult peaks seda muutma, tuleb siis, kui aru saan, mismoodi ma saan taski lisades isDone jätta undone ehk false ja selle nupu vajutamisel muuta nupu värvi, tõmmata kriips tekstile peale ja muuta isdone done'iks ja seeläbi lisada see tehtud taskide numbritesse +1'na ja selle nupu uuesti vajutamisel undo kõik, mis just ütlesin.

hhabakuk commented on script.js:
```diff
@@ -1,7 +1,7 @@
 const tasks = [
-    {title: 'Exercise 1', isDone: 'done'},
-    {title: 'Exercise 2', isDone: 'not done'},
+    {title: 'Exercise 1', isDone: 'true'},
```
kui on boolean, ära kasuta jutumärke - lihtsalt true / false

karmelp replied:
selle ma juba fixisin, jaa

karmelp committed:
my millionth logic on changing the value of task boolean

okei. ma olen mega palju katsetanud. Hetkel miskit sellist kirja jäänud. no mai tea mismoodi seda teha :D showtask funcis hakkab vähemalt disain muutuma, aga väärtust ma muudetud ei saa

hhabakuk commented on script.js:
```diff
@@ -87,17 +110,26 @@ function showNumberofDoneTasks() {
 // Kasutaja saab märkida ülesande tehtuks.
-
+// vajutades taskStatusEl (.taskStatusButton)
+document.querySelector('.taskStatusButton').onclick = function () {
```
`document.querySelector` tagastab sulle esimese elemendi, mis on `. taskStatusButton ` klassiga. Ehk praegu saab selle funkstiooni endale külge ainult esimene task. 

Kuna igal taskil peaks see funktsioon küljes olema, siis kasuta nt `document.querySelectorAll` meetodit - see tagastab sulle array kõikidest taskidest, mis sul DOMis on. Siis saad juba `forEachi` või `map` meetodiga üle käia kõik elemendid ning panna neile `onclick` külge.

hhabakuk replied:
enne kui edasi lähed taske märkima tehtuks/tegematuks, tee `console.log(task.title)`, et olla kindel, et saad kõik taskid kätte.

karmelp committed:
user can mark the task done

karmelp committed:
show done tasks count w toggle

karmelp committed:
Update script.js

karmelp committed:
removed taskstatus in text from task container
